index.html

```diff
@@ -67,5 +68,193 @@ <h1 class="social-media-heading">LET’S CONNECT</h1>
+            </li>
+          </ul>
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </section>
+      <!-- Fourth project-tonic -->
+      <section class="works">
+        <div class="tonic-1st">
+          <img
+            class="tonic-1st-img"
+            src="images/Snapshoot Portfolio-last.png"
+          />
+          <h1 class="tonic-1st-heading">Multi-Post Stories</h1>
+          <ul class="tonic-1st-list">
+            <li class="canopy">CANOPY</li>
+            <li class="list-vector-icon">
+              <i class="fa-solid fa-circle-small"></i>
+            </li>
+            <li class="backend">Back End Dev</li>
+            <li class="list-vector-icon">
+              <i class="fa-solid fa-circle-small"></i>
+            </li>
+            <li class="year">2015</li>
+          </ul>
+          <p class="works-passage">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="works-btns">
+            <li><button type="button">html</button></li>
+            <li class="works-btn"><button type="button">css</button></li>
+            <li class="works-btn">
+              <button type="button">javascript</button>
+            </li>
+          </ul>
+          <button class="see-project-btn" type="button">See Project</button>
+        </div>
+      </section>
+      <!-- The About Myself section -->
+      <section class="main-page-content">
+        <div class="primary-text">
+          <h1>About Myself</h1>
+        </div>
+        <p class="decsriptive-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h1 class="social-media-heading">LET’S CONNECT</h1>
+        <div class="parent-container">
+          <ul>
+            <li class="twitter">
+              <a href="#"><img src="images/twitter.png"</a>
+            </li>
+            <li class="linked-in">
+              <a href="#"
+                ><img class="linkedin-i" src="images/I.png" /><img
+                  src="images/N.png"
+              /></a>
+            </li>
+            <li class="message">
+              <a href="#"><img src="images/M.png" /></a>
+            </li>
+            <li class="cat-icon">
+              <a href="#"><img src="images/cat.png" /></a>
+            </li>
+            <li class="thumbs-up">
+              <a href="#"><img src="images/thumps.png" /></a>
+            </li>
+          </ul>
+        </div>
+        <div>
+          <button class="see-resume-btn" type="button">Get my resume</button>
+        </div>
+        <!-- Languages section -->
+        <div>
+          <h2 class="languages-h2">Languages</h2>
```

Comment:
- [ ] Kindly add the appropriate caret for this element. 👍 More so, I noticed that you did implement other technology cateories (Frameworks and skills); kindly add that as well. 😄